build.gradle.kts

```diff
@@ -6,12 +6,23 @@ plugins {
   kotlin("plugin.jpa") version "2.0.0"
   id("jacoco")
 }
-
+configurations {
+  implementation {
+    exclude(group = "org.xmlunit", module = "xmlunit-core")
+  }
+  testImplementation {
+    exclude(group = "org.xmlunit", module = "xmlunit-core")
+  }
+}
```

Comment:
Is this not working to exclude the dependencies from below?  If not is it needed? 

you can also mark as a false warning, e.g: [here](https://github.com/search?q=repo%3Aministryofjustice%2Fhmpps-kotlin-lib%20test-suppressions.xml&type=code)

Reply:
done